### ISSUE TITLE
Display zoom level map scale (e.g. 1:10,000) beneath scale bar when changing zoom

### ISSUE THREAD

**Issue by volcan01010** [MerginMaps/mobile#3698]
### Summary

As a geologist digitising lines via Mergin Input, I would like to know the absolute scale that I am working at so that I can digitise with an appropriate level of detail.

### Description

When are user zooms in or out in the Mergin Input user interface, a scale bar is shown that demonstrates how big a 50 m line (or whatever is appropriate).  After a while, the scale bar disappears again.

![Image](https://github.com/user-attachments/assets/8ae22a40-18ca-417c-ad6d-82333cd99774)

When geologists are digitising lines, they need to know the scale that they are working at so that they can mark vertices with an appropriate level of detail. If they are using a 1:10k or 1:25k Ordnance Survey basemap, the scale is easy to determine.  However, if they are using a DEM or hillshade, this is not possible.

Our geologists have requested an enhancement to the current scale bar implementation, which is to add another line beneath the scale bar that reports the zoom level e.g. 1:10,000.  This way they could zoom to a level that is appropriate.

They would not be able to hit a precisely defined level by pinching their fingers e.g. 1:10,124.  Rendering the zoom levels to just 2 significant figures would be good enough e.g. 1:150, 1:1500, 1:15k, 1:150k, 1:1.50M.

I don't know if 1:100,000 or 1:100k is easier to read or understand.

**Comment by VitorVieiraZ** [comment]
Thanks for the enhancement idea @volcan01010. It's a great suggestion, and we'll definitely consider it.

**Comment by volcan01010** [comment]
Great, thanks!

**Comment by saberraz** [comment]
A better way to handle this IMO is to have the scales set in QGIS project and then have a preset zoom button in the app to allow users to select the predefined scales:

![Image](https://github.com/user-attachments/assets/62002043-170a-42cd-ac2b-f3b5552587b7)

**Comment by volcan01010** [comment]
I agree that making pre-set map scales accessible in the Mergin App would also be a suitable way to allow users to know they were working at the correct scale.
